crates/codegen/runtime/cargo/wasm/src/runtime/interface/bindings.wit.jinja2

```diff
@@ -31,6 +31,9 @@ interface bindings {
         /// Returns the location of the definition's definiens.
         /// For `contract X {}`, that is the location of the parent `ContractDefinition` node.
         definiens-location: func() -> binding-location;
+
```

Comment:
Copying the earlier suggestion about identifiers here:

> WDYT of expanding on the doc comments above `resource binding-graph {}` here, to indicate that some identifiers (imports, aliases, anything else?) can be both definitions and references, and how can users handle these cases?

Not sure if it is worth creating a separate PR for it.

Reply:
No, I'll try to expand the docs on this PR. Thanks for the nudge!